app/models/series.rb

```diff
@@ -37,6 +37,7 @@ def description_md
   has_many :images, -> { where(parent_id: nil) }, class_name: 'SeriesImage', dependent: :destroy
 
   before_validation :set_app_version, on: :create
+  before_validation :update_account_for_stories, on: :update
```

Comment:
I tend not to make changes to child models in `before` operations. `before_validation` will run even if the change to the series doesn't save, or if `valid?` is called on the model and no save will occur. I think I would do this either in `after_save`, or `after_commit`